lib/config/parameters_parser.js

```diff
@@ -170,7 +179,7 @@ class RandomizeConfigurationParameterParser {
 class LanguageConfigurationParameterParser {
```

Comment:
I'd like to propose a refactor:

`LanguageConfigurationParameterParser`, `DirectoryConfigurationParameterParser` and `FileExtensionConfigurationParameterParser` only differ on data, not behavior. So they can be objects of the same class instead of separate classes. It can be an object that receives the param identifiers `['-l', '--language']` and the property name for building the configuration (`directory`, `filter` or `language`)